dirs/dirs.go

```diff
@@ -33,6 +36,8 @@ import (
 
 // the various file paths
 var (
+	stderr io.Writer = os.Stderr
```

Comment:
maybe this should be its own var/stanza, it doesn't match what is in the doc comment for this stanza

Reply:
this is no longer needed.